src/server/controller/geo/getForestLayer.ts

```diff
@@ -27,7 +27,7 @@ export const getForestLayer = async (props: Props): Promise<Layer> => {
       .getMap(
         {
           palette: metadata.palette,
-          opacity: opacity === undefined ? 1 : Number(opacity),
+          opacity: opacity === undefined ? 1 : opacity,
```

Comment:
can this be
`opacity: opacity ?? 1`

Reply:
Sure, I was not familiar with that operator :)

Reply:
I removed this control in last commit because I am setting default values when validating/sanitizing request params